src/EditorFeatures/CSharpTest/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.VisualStudio.Composition": "14.2.19-pre",
```

Comment:
I wonder how many of these are redundant and already present due to transitivity.

Also - does the 15.0 version work without updating the rest of our dependencies to 15.0?